src/picasso-definition/components/mini-chart/mini-points.js

```diff
@@ -0,0 +1,69 @@
+import KEYS from '../../../constants/keys';
+import NUMBERS from '../../../constants/numbers';
+
+export default function createMiniChartPoints(chartModel) {
+  const ratio = NUMBERS.MINI_CHART.RATIO;
+  const padding = NUMBERS.MINI_CHART.PADDING; // Padding from the bottom right corner
+
+  // Padding, normalized to chart size
+  let px;
+  let py;
```

Comment:
Put these in an object instead:
```
const padding = {
  x: 0,
  y: 0,
}
```
Reduced the need for the comment. Will also make it a lot clearer when they are used (`padding.x`) what they mean.

Same goes for home state and bin variables below.

It's always good to try to use `let` as little as possible in general, and good to try to group properties together in objects.